cognite/powerops/custom_modules/power_model_v1/data_models/containers/ShopScenario.container.yaml

```diff
@@ -37,14 +37,22 @@ properties:
     autoIncrement: false
     name: source
     description: The source of the scenario
-indexes:
-  nameIndex:
-    properties:
-      - name
-    indexType: btree
-    cursorable: false
-  sourceIndex:
-    properties:
-      - source
-    indexType: btree
-    cursorable: false
+    # outputDefinition:
+    #   type:
+    #     list: true
+    #     type: direct
+    #   nullable: true
+    #   autoIncrement: false
+    #   name: outputDefinition
+    #   description: The output definition
```

Comment:
Why is this commented out? Not needed in container because it is a list of edges?